src/cljs/witan/ui/controllers/datastore.cljs

```diff
@@ -951,6 +969,16 @@
 
   (.info js/toastr (gstring/format (get-string :stringf/file-deleted) (get-in args [:context :name]))))
 
+(defmethod on-activity-finished
+  [:add-collect-files-to-datapack :completed]
+  [{:keys [args]}]
+  (.info js/toastr (:log args)))
+
+(defmethod on-activity-finished
+  [:add-collect-files-to-datapack :failed]
+  [{:keys [args]}]
+  (.info js/toastr (:log args)))
+
```

Comment:
Once the activity finishes, what happens? What does the user see or where do they go?